Tests/ConnectionPoolModuleTests/Mocks/MockConnection.swift

```diff
@@ -97,6 +97,17 @@ final class MockConnection: PooledConnection, Sendable {
             callback(nil)
         }
     }
+
+    var isClosing: Bool {
+        self.lock.withLockedValue { state in
+            switch state {
+            case .closing:
+                return true
+            case .running, .closed:
+                return true
```

Comment:
this looks wrong ;)

Reply:
Of course it is :), I've removed it as it's no longer needed